### PR TITLE
Updating app json

### PR DESCRIPTION
**Pull request by itsjoekent**
### What does this PR do?
- Added keys that were missing
- Alphabetizing everything
- Adding the Redis Addon

### Any background context you want to provide?
Getting Heroku staging up to date with everything

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -2,19 +2,36 @@
   "name": "phoenix-next",
```
Comment (weerd) [inline app.json]: does the name need to change?

Comment (itsjoekent) [reply app.json]: 👍

**Comment by itsjoekent** [comment]
also just for anyone casually following -- I mentioned to Diego IRL we could add environment specific stuff here if we want, but right now we're only relying on the app.json for the review apps so not a huge deal

**Comment by itsjoekent** [comment]
screen shot 2017-11-16 at 1 56 26 pm

seems like it worked, from the review app on this PR ^

**Pull request merged**